selfdrive/controls/lib/pid.py

```diff
@@ -27,6 +27,7 @@ def __init__(self, k_p, k_i, k_f=1., pos_limit=None, neg_limit=None, rate=100, s
 
     self.sat_count_rate = 1.0 / rate
     self.i_unwind_rate = 0.3 / rate
+    self.i_bf_rate = 2.0 / rate
```

Comment:
Unwind rate would work better here for clarity but it's used in the old logic still